test/rust/basic/add_overflow.rs

```diff
@@ -0,0 +1,11 @@
+#[macro_use]
+mod smack;
+use smack::*;
+// @flag --integer-overflow
```

Comment:
Add empty line before this line. Make all of the regressions consistent like that.